packages/esm-form-entry-app/package.json

```diff
@@ -87,7 +89,7 @@
     "karma-coverage-istanbul-reporter": "~3.0.2",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "openmrs": "next",
+    "openmrs": "4.0.2-pre.336",
```

Comment:
Why not keep the generic `next`? Right now it would be [4.0.3-pre.441](https://www.npmjs.com/package/openmrs/v/4.0.3-pre.441).

Reply:
oh I forgot that I will fix it ASAP.